### DISCUSSION TITLE
High resource usage using Ditto on VMware

### DISCUSSION THREAD

**Discussion by mihau1211** [eclipse-ditto/ditto#1846] [Q&A]
Hi,
I had an issue with using Ditto by Docker. After I run docker-compose I have usage of CPU at 100%. The VM is using c.a, 90-95%.

Ubuntu VMware config:
![image](https://github.com/eclipse-ditto/ditto/assets/73185784/7aeebabc-9de8-4475-9147-1bc80fc0fb98)

My PC:
![image](https://github.com/eclipse-ditto/ditto/assets/73185784/80a0b7ac-4e76-47c8-947e-966f879128ed)

Is this a common issue or is there something wrong with my settings?

**Comment by thjaeckle** [comment]
Hard to tell with that few information.

A just started Ditto should not take more than 5% CPU ..
